js/try.js

```diff
@@ -261,33 +353,34 @@ const fetchAndParseTimetable = async (url) => {
       )
       .join(', ')
 
+    const startHour = extractTime(info[7].text);
+    const endHour = extractTime(info[8].text);
+
+    const day = new Date(date);
+    day.setDate(day.getDate() + dayIndex);
+
+    const startDate = new Date(day);
+    startDate.setHours(startHour);
```

Comment:
With the above `date` field, you wouldn't need to use `startHour` and `endHour`.

Reply:
turns out you don't need them anyway - i kept them cos i thought you code uses them, but it turns out it doesn't

removing.

Reply:
you mean `start: startHour`? it seems that your code is using this field.